resources/serverless/templates/virtualservice.yaml

```diff
@@ -1,3 +1,4 @@
+{{/*TODO remove it*/}}
```

Comment:
why not to remove it now in this PR ?
Having this makes istio still required

Reply:
I have another PR with istio deletion: https://github.com/kyma-project/kyma/pull/17257/files